README.md

```diff
@@ -682,6 +682,24 @@ middleware.NewLogger = func() (*zap.Logger, error) {
 }
 ```
 
+### Getting Operation Info
```

Comment:
I like having the how-to example, any chance we could also add a table to the docs that breaks down every piece of data that's in a request context object and what function to call to pull it out?  Doesn't need to be in this PR.

Reply:
Sure, it'll change based on middleware but we can definitely document what is present by default and which built-in middlewares add to the context.

Reply:
Maybe that info is more appropriate in end services then?  That is, ChannelAPI listing out every middleware its running and what it's dumping into the context object?  As an end user it's property bag without a schema and it's not always clear what's in it or how to get it.